hail/src/test/scala/is/hail/expr/ir/TableIRSuite.scala

```diff
@@ -427,6 +427,35 @@ class TableIRSuite extends HailSuite {
     assert(beforeValue.rdd.collect().toFastIndexedSeq == after.rdd.collect().toFastIndexedSeq)
   }
 
+  @Test def testTableRenameLowering() {
+    implicit val execStrats = ExecStrategy.allRelational
```

Comment:
Note: This is just the default set of execution strategies for this test suite. I went through and explicitly defined the strategies for the test methods that couldn't be lowered, with the idea being that we could just take them out as we lowered, so that it's easy to see which tests work when lowered and which ones don't.

Reply:
Also, the test above this (testTableRename) should now work through the lowerer, although it's currently written in such a way that it's not going through the test eval stuff at all. I would have a slight preference for combining both tests into one test that goes through the lowerer, since they look basically the same except that one goes through assertEvalsTo and one does a check on the type of the renamed table.

Reply:
Yeah the reason I left the first one is because it was going through `Interpret` explicitly and then was checking the type of the `TableValue`. I didn't know how to emulate that since I don't get a `TableValue` through lowering as far as I'm aware. I could just get rid of the first test? The new one seems more thorough.